libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -71,10 +83,57 @@ func (p *addHostMetadata) loadData() {
 		if p.info.OS.Build != "" {
 			p.data.Put("host.os.build", p.info.OS.Build)
 		}
+
+		if p.config.NetInfoEnabled {
+			// IP-address and MAC-address
+			var ipList, hwList = p.getNetInfo()
+			p.data.Put("host.ip", ipList)
```

Comment:
Reviewing the code below and the possibility that ipList and hwList can be empty, we should check for `len(x) == 0` and in case of 0 not add the field.

Reply:
Ok, I'll fix that